templates/skeleton/app/routes/account/login.tsx

```diff
@@ -18,13 +16,7 @@ export async function loader({context, params}: LoaderArgs) {
   return new Response(null);
 }
 
-type ActionData = {
-  formError?: string;
-};
-
-const badRequest = (data: ActionData) => json(data, {status: 400});
-
-export const action: ActionFunction = async ({request, context, params}) => {
+export const action: ActionFunction = async ({request}) => {
```

Comment:
Yeah I didn't end up including that section in the guidelines because there were still some questions around it.

Additionally, it seems that `ErrorBoundaryType` doesn't actually work with function declarations. Or at least, I can't get it to work. So, 🤷